Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -31,39 +26,23 @@ public enum AgentRole
     public Rigidbody agentRb;
     SoccerSettings m_SoccerSettings;
     Renderer m_AgentRenderer;
+    BehaviorParameters m_BP;
+    Vector3 m_Transform;
 
-    public void ChooseRandomTeam()
+    public override void InitializeAgent()
     {
-        team = (Team)Random.Range(0, 2);
-        if (team == Team.Purple)
+        base.InitializeAgent();
+        m_BP = gameObject.GetComponent<BehaviorParameters>();
+        if (m_BP.m_TeamID == 0)
```

Comment:
nit: could use explicitly assign these values [0, 1] to the `Team` enum and compare against that?